app/views/presentations/index.html.erb

```diff
@@ -0,0 +1,29 @@
+<h1>Presentations</h1>
+
+<%= link_to 'New Presentation', new_presentation_path %>
+
+<table>
+  <thead>
+    <tr>
+      <th>Presentation</th>
+      <th>
+    </tr>
+  </thead>
+  <tbody>
+    <% @presentation.each do |presentation| %>
+      <tr>
+        <td><%= presentation.topic %></td>
+        <td>
+          <%= link_to 'Show', presentation_path(presentation) %>
+          <%= link_to 'Edit', edit_presentation_path(presentation) %>
+          <%= link_to 'Delete', presentation_path(presentation),
+                      method: :delete,
+                      data: { confirm: 'Are you sure?' } %>
```

Comment:
It just puts all three links in one line. It doesn't look really great, and I'm not sure that we should be using tables at all, but as mentioned above, I used the people index view as a template for the time being until we have decided on format, flow, and styling for views.